docs.md

```diff
@@ -0,0 +1,1366 @@
+ggplot(breast_cancer_data, mapping = aes(y=Glucose, x=Age)) +
+  geom_abline(intercept = model$coefficients[1], slope = model$coefficients[2]) +
+  scale_y_continuous(limits = c(min(breast_cancer_data$Glucose), max(breast_cancer_data$Glucose))) +
+  scale_x_continuous(limits = c(min(breast_cancer_data$Age), max(breast_cancer_data$Age))) +
+  theme_bw()
+```
+```r  -Solution using geom_line
+ggplot(breast_cancer_data, mapping = aes(y=Glucose, x=Age)) +
+  geom_line(mapping = aes(y=model$fitted.values)) +
+  theme_bw()
+```
+```r  -Another solution, using alpha
+# you can also make any element of a plot invisible by setting its alpha to 0
+# in this case, we can make the dots of the scatterplot disappear from any of the plots we made above
+# for example:
+ggplot(breast_cancer_data, mapping = aes(y=Glucose, x=Age)) +
+  geom_point(alpha = 0) +
+  geom_line(mapping = aes(y=model$fitted.values)) +
+  theme_bw()
+
+# this has the advantage of keeping the scales for the plot consistent
+# and it means you don't have to set the scales manually when using geom_abline
+```
+````
+
+Write code to draw a linear trend line showing the relationship between Age and Glucose, but create a plot with just the line, no scatterplot underneath. Try it each of the three ways, using geom\_smooth, geom\_abline, and geom\_line.
+
+```r  -Solution using geom_smooth
+ggplot(breast_cancer_data, mapping = aes(y=Glucose, x=Age)) +
+  geom_smooth(method = "lm") +
+  theme_bw()
+```
+```r  -Solution using geom_abline
+# note that this doesn't actually plot a line, since there are no observations to set the x and y scales
+# you'll see a blank plot
+ggplot(breast_cancer_data, mapping = aes(y=Glucose, x=Age)) +
+  geom_abline(intercept = model$coefficients[1], slope = model$coefficients[2]) +
+  theme_bw()
+
+# you can set the x and y scales yourself manually by adding a layer for each
+ggplot(breast_cancer_data, mapping = aes(y=Glucose, x=Age)) +
+  geom_abline(intercept = model$coefficients[1], slope = model$coefficients[2]) +
+  scale_y_continuous(limits = c(min(breast_cancer_data$Glucose), max(breast_cancer_data$Glucose))) +
+  scale_x_continuous(limits = c(min(breast_cancer_data$Age), max(breast_cancer_data$Age))) +
+  theme_bw()
+```
+```r  -Solution using geom_line
+ggplot(breast_cancer_data, mapping = aes(y=Glucose, x=Age)) +
+  geom_line(mapping = aes(y=model$fitted.values)) +
+  theme_bw()
+```
+```r  -Another solution, using alpha
+# you can also make any element of a plot invisible by setting its alpha to 0
+# in this case, we can make the dots of the scatterplot disappear from any of the plots we made above
+# for example:
+ggplot(breast_cancer_data, mapping = aes(y=Glucose, x=Age)) +
+  geom_point(alpha = 0) +
+  geom_line(mapping = aes(y=model$fitted.values)) +
+  theme_bw()
+
+# this has the advantage of keeping the scales for the plot consistent
+# and it means you don't have to set the scales manually when using geom_abline
+```
```

Comment:
Would it make sense to have a feedback form at the end of this to actually get feedback from anyone who tries to use it?
I know the feedback macro will be in the templates, and people will be unlikely to use this documentation linearly with an ending point where they want to give feedback, but we might get some good user insights if we ask for them.

Reply:
I like this idea. I think we'd want a different survey for it, though, right? We'd want slightly different questions than what we ask in the quick module feedback form. 
I've been wondering about whether we want to have a Quick General Feedback survey, for folks to submit anonymous suggestions that aren't tied to a particular module. This could maybe fit into something like that.

Reply:
Oooh, I really like the idea of a quick general feedback survey, like for people who have already filled out the interest form and have some general thing to tell us :)